pos_product_label/models/pos_session.py

```diff
@@ -32,6 +32,7 @@ def print_product_labels(self, data):
         """
         vals = self._prepare_product_label_layout_data(data)
         wizard = self.env["product.label.layout"].create(vals)
+        wizard = wizard.with_context(pos_config_id=self.config_id)
```

Comment:
Where this variable `pos_config_id` is going to be used ? I don't see any usage in module `pos_product_label`

Reply:
Neither in Odoo core 🤔

Reply:
If what you need is to add a custom context key used in an external module, you can always override `print_product_labels` like this, in your custom module:

```py
def print_product_labels(self):
    self = self.with_context(pos_config_id=self.config_id)
    return super().print_product_labels()
```